client/payment-details/dispute-details/dispute-awaiting-response-details.tsx

```diff
@@ -70,7 +70,7 @@ const DisputeAwaitingResponseDetails: React.FC< Props > = ( {
 				<CardBody className="transaction-details-dispute-details-body">
 					<DisputeNotice
 						dispute={ dispute }
-						urgent={ countdownDays <= 2 }
+						isUrgent={ countdownDays <= 2 }
```

Comment:
I've renamed this to isUrgent so it's super obvious it's a boolean. I'm not sure if we have a relevant guideline but this is common in WP code (is my understanding). 

I saw `isDismissible` [in block editor JS guidelines](https://developer.wordpress.org/block-editor/contributors/code/coding-guidelines/) so this seems aligned with convention.